packages/vite/src/ember.ts

```diff
@@ -0,0 +1,52 @@
+import { templateTag } from './template-tag.js';
+import { resolver } from './resolver.js';
+import { mergeConfig, type UserConfig } from 'vite';
+import { esBuildResolver } from './esbuild-resolver.js';
+
+export let extensions = ['.mjs', '.gjs', '.js', '.mts', '.gts', '.ts', '.hbs', '.hbs.js', '.json'];
+
+export function ember() {
+  return [
+    templateTag(),
+    resolver(),
+    {
+      name: 'vite-plugin-ember-config',
+      async config(config: UserConfig, env: { mode: string; command: string }) {
```

Comment:
a type of ConfigEnv exists in vite, can use that for the env argument.

```ts
import { type ConfigEnv } from 'vite';
```

Reply:
nice one thanks!